TF_Service_and_Tooling_Unity/Assets/TouchFree/ServiceUI/Prefabs/ConfigCanvas.prefab

```diff
@@ -1819,11 +1819,6 @@ PrefabInstance:
   m_Modification:
     m_TransformParent: {fileID: 4852224622988348253}
     m_Modifications:
```

Comment:
This prefab has a broken script on the root when looking at the prefab itself, the instance in the scene seems fine though